lib/jellyfish/component/file.ex

```diff
@@ -35,7 +38,14 @@ defmodule Jellyfish.Component.File do
 
       properties = valid_opts |> Map.from_struct()
 
-      {:ok, %{endpoint: endpoint_spec, properties: properties}}
+      new_framerate =
+        if track_config.type == :video do
+          properties.framerate || 30
+        else
+          properties.framerate
+        end
```

Comment:
Since we already set the framerate in `get_track_config`:
```suggestion
        if track_config.type == :video do
          track_config.framerate
        else
          nil
        end
```